frontend/src/app/pg-flow-reg-person-friend-family/pg-flow-reg-person-friend-family.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-pg-flow-reg-person-friend-family',
+  templateUrl: './pg-flow-reg-person-friend-family.component.html',
+  styleUrls: ['./pg-flow-reg-person-friend-family.component.scss']
```

Comment:
for angular is is necessary to have .scss for each page? seems kind of inefficient.